src/hooks/useIsRewardClaimed.ts

```diff
@@ -0,0 +1,12 @@
+import { RolePlatform } from "../types"
+import { useGetUserRewards } from "./useGetUserRewards"
+
+export const useIsRewardClaimed = (platformId: RolePlatform["id"]) => {
+  const { isLoading, data: userRewards } = useGetUserRewards()
+
+  const claimed = !isLoading
```

Comment:
I think we don't need this ternary here, this expression will return either a reward or `undefined` anyways